### DISCUSSION TITLE
【任务】8月社区激励任务｜20,000 $PEOPLE

### DISCUSSION THREAD

**Discussion by AmazingAng** [AmazingAng/WTF-Solidity#174] [任务 | Bounty]
从8月开始，WTF学院每月的社区激励上调到20,000 $PEOPLE，这包括日常的PR，也包括审核、改善WTF极简教程。这部分奖励优先分配给月度社区激励任务，剩余分给当月贡献者。

## 任务1：审核WTF Solidity 41-46讲，并补充图片

**赏金**：3,000 $PEOPLE 代币，根据工作量分配。

### 认领情况：
第41讲：WETH 
@seasidejuvenile666   **已完成** #161 
第42讲：分账 
@Big-Aaron **已完成** #157 
第43讲：线性释放
@Big-Aaron **已完成** #160 
第44讲：代币锁
@seasidejuvenile666  **已完成** https://github.com/AmazingAng/WTFSolidity/pull/175
第45讲：时间锁
@chad779  **已完成** #185 
第46讲：代理合约
@seasidejuvenile666 **已完成** #190

## 任务2：审核WTF Ethers 1-5讲，并补充图片

**赏金**：2,500 $PEOPLE 代币，根据工作量分配。

第1讲：HelloVitalik（6行代码）：【[代码](https://github.com/WTFAcademy/WTFEthers/blob/main/01_HelloVitalik)】 【[教程](https://github.com/WTFAcademy/WTFEthers/blob/main/01_HelloVitalik/readme.md)】
@beihaili
第2讲：Provider 提供器：【[代码](https://github.com/WTFAcademy/WTFEthers/blob/main/02_Provider)】 【[教程](https://github.com/WTFAcademy/WTFEthers/blob/main/02_Provider/readme.md)】
@beihaili
第3讲：读取合约信息：【[代码](https://github.com/WTFAcademy/WTFEthers/blob/main/03_ReadContract)】 【[教程](https://github.com/WTFAcademy/WTFEthers/blob/main/03_ReadContract/readme.md)】
@zhiyuan2007 **已完成** https://github.com/WTFAcademy/WTFEthers/pull/9
第4讲：发送ETH：【[代码](https://github.com/WTFAcademy/WTFEthers/blob/main/04_SendETH)】 【[教程](https://github.com/WTFAcademy/WTFEthers/blob/main/04_SendETH/readme.md)】
@zhiyuan2007 **已完成** https://github.com/WTFAcademy/WTFEthers/pull/9
第5讲：合约交互：【[代码](https://github.com/WTFAcademy/WTFEthers/blob/main/05_WriteContract)】 【[教程](https://github.com/WTFAcademy/WTFEthers/blob/main/05_WriteContract/readme.md)】
@seasidejuvenile666  **已完成** https://github.com/WTFAcademy/WTFEthers/pull/5

## 任务3：翻译WTF Solidity 11-15讲

**赏金**：5,000 $PEOPLE 代币，根据工作量分配。

**要求**：遇到Solidity的名词要查询[官方英文文档](https://docs.soliditylang.org/en/v0.8.13/introduction-to-smart-contracts.html)

第11讲：构造函数（constructor）和修饰器（modifier）
@seasidejuvenile666 **已完成** https://github.com/AmazingAng/WTFSolidity/pull/175
第12讲：事件（events）
@seasidejuvenile666 **已完成** https://github.com/AmazingAng/WTFSolidity/pull/175
第13讲：继承
@quantum-proof
第14讲：抽象合约（abstract）和接口（interface）
@JustinAsdz **已完成** #186
第15讲：异常（errors）
@JustinAsdz **已完成** #191

## 认领方式
在这个帖子下面回复就能认领（目前1人每个任务最多认领2讲，完成后可认领新的）。

有用的资料：
1. Markdown书写规则（**必看**）：https://www.markdownguide.org/cheat-sheet/
2. 审核并添加图片后的例子：https://github.com/AmazingAng/WTFSolidity/tree/main/02_ValueTypes
3. 审核第一期：https://github.com/AmazingAng/WTFSolidity/discussions/18
4. https://github.com/AmazingAng/WTFSolidity/discussions/132

**Comment by seasidejuvenile666** [comment]
认领solidity44讲、ethers第5讲、翻译11、12讲

**Reply by AmazingAng** [reply]
好的

**Reply by seasidejuvenile666** [reply]
继续认领solidity45、46讲

**Reply by AmazingAng** [reply]
好的

**Comment by quantum-proof** [comment]
翻译 13

**Reply by chad779** [reply]
认领solidity45讲，时间锁。

**Reply by AmazingAng** [reply]
o k

**Reply by AmazingAng** [reply]
> 认领solidity45讲，时间锁。

chad779 十天没有更新 也没用填表加入社区 默认你放弃任务了

**Comment by JustinAsdz** [comment]
翻译14，15讲

**Reply by AmazingAng** [reply]
ok

**Reply by JustinAsdz** [reply]
14: #186

**Comment by hhsmiku77** [comment]
Solidity - 46 讲 , Ethers - 1 & 2 讲 Please...

**Reply by AmazingAng** [reply]
ok

**Reply by zhiyuan2007** [reply]
认领
第3讲：读取合约信息
第4讲：发送ETH

**Reply by beihaili** [reply]
> 十天没有更新 也没用填表加入社区 默认你放弃任务了

接替认领 ethers -1 -2

**Reply by AmazingAng** [reply]
好 我看错了 3、4更新了 1、2没更新，你认领1、2吧

**Reply by beihaili** [reply]
> 好 我看错了 3、4更新了 1、2没更新，你认领1、2吧

好的

**Comment by Big-Aaron** [comment]
这是来自QQ邮箱的自动回复邮件。你好，你的邮件我已经收到。
